gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionSha256Sum=13bf8d3cf8eeeb5770d19741a59bde9bd966dd78d17f1bbad787a05ef19d1c2d
```

Comment:
Why did we remove the sha256 checksum?

Checksum for 7.1.1 is `9bb8bc05f562f2d42bdf1ba8db62f6b6fa1c3bf6c392228802cc7cb0578fe7e0` FYI
REF: https://gradle.org/release-checksums/

Reply:
Good point. The wrapper task did that.